lib/chef/knife/azurerm_base.rb

```diff
@@ -141,56 +136,49 @@ def is_token_valid?(token_details)
           return false
         elsif time_difference <= 600 # 600sec = 10min
           # This is required otherwise a long running command may fail inbetween if the token gets expired.
-          raise "Token will expire within 10 minutes. Please run 'azure login' command"
+          raise_azure_timeout
```

Comment:
I guess we should only make method only if there is a code repetition.

Reply:
done